### PR TITLE
Header updates per UX review

### PR DESCRIPTION
**Pull request by matthopson**
Style updates per UX review

### CODE REVIEW

**Comment by wbyoko** [comment]
Should this be onto develop?

File: web-client/src/styles/custom.scss [hunk 1]
```diff
@@ -623,6 +629,7 @@ nav.usa-nav {
 
   &.mobile-menu {
     background: $color-cool-blue;
+    font-family: 'Source Sans Pro', sans-serif;
```
Comment (kkoskelin) [inline web-client/src/styles/custom.scss]: Do we not have a `$font-something` variable for this?  I think these will need to be changed.

**Review by kkoskelin — CHANGES_REQUESTED** [review]

File: web-client/src/styles/custom.scss (hunk 1 above)
Comment (matthopson) [reply web-client/src/styles/custom.scss]: We do not.

**Comment by matthopson** [comment]
@wbyoko I'm not sure I follow. Some header changes already exist in `develop` - this is an update to those before we demo tomorrow.

**Comment by wbyoko** [comment]
I didn't know if the BEM notation was being used before 2.0. okay.

**Review by sutt0n — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]
Wondering if there was another variable already declared, but this is a step in the right direction.  let's do it!

**Pull request merged**